src/Controls/tests/TestCases.HostApp/Issues/Bugzilla/Bugzilla31366.cs

```diff
@@ -32,7 +32,7 @@ public MainPage()
 
 				popOnAppearing.Appearing += async (sender, args) =>
 				{
-					await Task.Yield();
+					await Task.Delay(50);
```

Comment:
Could just include a View, like a Label, and wait for it?

Reply:
In the test case host app, a new page is being pushed and then immediately popped on the page's Appearing event. As a result, the page's UI does not load, making it impossible to validate views like labels. Here, added a slight delay before popping the page in the Appearing event to avoid exception.